src/ex1/style.css

```diff
@@ -0,0 +1,122 @@
+* {
+	margin: 0;
+}
+body {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	/* height: 100vh; */
+	background: linear-gradient(135deg, rgb(119, 75, 238), rgb(190, 59, 230));
+	font-family: system-ui;
+}
+
+h1 {
+	font-weight: 700;
+	color: rgb(190, 59, 230);
+}
+
+.container {
+	position: relative;
+	border: 1px solid rgba(142, 158, 174, 0.701);
+	border-radius: 8px;
+	padding: 1rem;
+	background-color: aliceblue;
+	width: 300px;
+}
+
+.newTaskContainer {
+	position: relative;
+	display: flex;
+	justify-content: row;
+	align-items: center;
+	margin-bottom: 20px;
+}
+
+.task-input {
+	width: 85%;
+	height: 50px;
+	border: none;
+	border-radius: 5px;
+	font-size: 15px;
+	font-weight: 700;
+	margin-right: 4px;
+	background-color: rgba(127, 255, 212, 0.597);
+}
+
+.taskElement {
+	background-color: rgba(223, 223, 223, 0.508);
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 10px;
+	height: 30px;
+	max-width: 300px;
+	border-radius: 8px;
+	padding-left: 1rem;
+	overflow: hidden;
+	font-size: 15px;
+	font-weight: 700;
+}
+
+.taskElement:hover {
+	overflow-y: hidden;
+	overflow-x: hidden;
+	height: max-content;
+}
+
+.clickable {
+	cursor: pointer;
+}
+
+.delete-button {
+	background-color: rgba(205, 89, 89, 0.841);
+	display: none;
+	width: 30px;
+	animation-name: elemet-size;
+	animation-duration: 0.2s;
+	border-top-right-radius: 8px;
+	border-bottom-right-radius: 8px;
+}
+
+.clear-all {
+	display: none;
+	float: right;
+}
+
+.tash-icon {
+	padding: 5px;
+	width: 20px;
+	height: 20px;
+	animation-name: img-size;
+	animation-duration: 0.2s;
+}
+
+.pluse-icon {
+	width: 20px;
+	height: 20px;
+	border-radius: 50%;
+	padding: 5px;
+	background-color: rgba(127, 255, 212, 0.597);
+}
+
+.amountTasks {
```

Comment:
should keep naming convention of classes - See 'Best Practices' comment, section 1

Same for other classes in this file